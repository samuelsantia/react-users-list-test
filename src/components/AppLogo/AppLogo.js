import styled from 'styled-components/macro';

/** Renders App Logo */
const AppLogo = styled.h1`
  --font-increase: var(--app-logo-font-increase, 3);
  --rhythm-increase: var(--app-logo-rhythm-increase, 2);
  text-transform: uppercase;
  font-weight: lighter;
  color: var(--app-logo-base-color, inherit);

  strong {
    font-weight: bold;
    color: var(--app-logo-strong-color, inherit);
  }
`;

export default AppLogo;
