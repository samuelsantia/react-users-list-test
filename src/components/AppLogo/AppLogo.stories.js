import React from 'react';

import AppLogo from './AppLogo';

export default {
  title: 'AppLogo',
  component: AppLogo,
};

export const basic = () => (
  <AppLogo>
    <strong>App</strong>Logo
  </AppLogo>
);
