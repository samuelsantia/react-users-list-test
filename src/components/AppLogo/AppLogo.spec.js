import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import AppLogo from './AppLogo';

describe('<AppLogo />', () => {
  test('it should render', () => {
    const { container } = render(
      <AppLogo>
        <strong>Test</strong>Logo
      </AppLogo>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
