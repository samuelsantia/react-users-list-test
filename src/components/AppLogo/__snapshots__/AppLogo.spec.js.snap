// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLogo /> it should render 1`] = `
.c0 {
  --font-increase: var(--app-logo-font-increase,3);
  --rhythm-increase: var(--app-logo-rhythm-increase,2);
  text-transform: uppercase;
  font-weight: lighter;
  color: var(--app-logo-base-color,inherit);
}

.c0 strong {
  font-weight: bold;
  color: var(--app-logo-strong-color,inherit);
}

<h1
  class="c0"
>
  <strong>
    Test
  </strong>
  Logo
</h1>
`;
